### DISCUSSION TITLE
Any advice or guides to get WSS traffic to show in Homer?

### DISCUSSION THREAD

**Discussion by PadScratchJohns** [sipcapture/homer#627] [Q&A] [answered]
Hey all, I have a set of Kamailio's that do WebRTC <-> SIP conversion and I want to log messaging for these interactions for debugging etc. 

On SIP -> WebRTC I see these calls no issue, but the other way I do not. So initiated from browser to core I get nothing in Homer. 

Calls work in both directions and certs are on the box - calls in the other direction (inbound call from core to WebRTC) log correctly and show in Homer.

I am using websocket module in Kamailio for this and the correct params are in the siptrace settings. 

Pretty sure I have missed something, but cannot find anything around capturing websocket or secure websocket traffic in the wiki. 

Can anyone point it out in the wiki (or a handy URL) for me? 

Cheers,

John.

**Comment by hamidrhashmi** [comment]
Use [rtcagent](https://github.com/sipcapture/rtcagent) and it will capture WebRTC traffic as well.

**Comment by adubovikov** [comment, marked as answer]
@PadScratchJohns the same way as usual - activate siptrace module and duplicate all messages to HOMER

**Reply by PadScratchJohns** [reply]
This is the answer, it was a config issue on my side.

**Reply by gary391** [reply]
Can you share your configure example ?
